docs/source/routing/self-hosted/containerization/kubernetes/other-considerations.mdx

```diff
@@ -56,3 +58,13 @@ If your deployment of routers is terminating due to memory pressure, you can add
 Apollo Router stops accepting new requests immediately after receiving a `SIGTERM` in Kubernetes. It waits for all active requests to complete before shutting down. There is no time limit controlling this process; it continues until all requests either finish or time out at the request level.
 
 You can configure health checks using [Kubernetes lifecycle hooks](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/) to ensure all pods shut down safely. You can also set the shutdown grace period to be slightly longer than any configured [router timeouts](/graphos/routing/performance/traffic-shaping).
+
+## Using the Apollo GraphOS Operator
+
+  For more advanced Kubernetes deployments, consider using the [Apollo GraphOS Operator](https://www.apollographql.com/docs/apollo-operator/) to manage your router and subgraph lifecycle. The Operator provides declarative Kubernetes resources for managing supergraphs and can simplify complex multi-service
```

Comment:
```suggestion
  For more advanced Kubernetes deployments, consider using the [Apollo GraphOS Operator](/apollo-operator) to manage your router and subgraph lifecycle. The Operator provides declarative Kubernetes resources for managing supergraphs and can simplify complex multi-service
```
Our style guide recommends using absolute paths rather than URLs.